packages/wonder-blocks-tooltip/components/tooltip-tail.js

```diff
@@ -273,38 +288,51 @@ export default class TooltipArrow extends React.Component<Props> {
     }
 }
 
+/**
+ * Some constants to make styles easier to understand.
+ * NOTE: The widths and heights refer to the downward-pointing tail
+ * (i.e. placement="top"). When the tail points to the left or right instead,
+ * the width/height are inverted.
+ */
```

Comment:
These are awesome, thanks!

Reply:
Also, thanks for describing what happens with these values for the different placements.

Reply:
Thanks. I am changing these a bit since we are going to have different padding for left/right and top/bottom (see zeplin). I'll keep it all nice and documented though.